### DISCUSSION TITLE
BUG

### DISCUSSION THREAD

**Discussion by ylq1** [0Quake/Zero-Quake#24] [General]
コンピュータの時刻がUTC+8の場合、受信した情報がUTC+9であっても、Zero Quakeの時刻もUTC+8に設定され、Zero Quakeは1時間前の情報を受信します。 例えば、現在のUTC+8の時刻が14:00の場合、Zero Quakeの時刻も14:00に変更されますが、日本の正しい時刻は実際には15:00です

**Comment by 0Quake** [comment]
バグのご報告ありがとうございます。こちらの環境で再現し、問題を確認しました。
JST以外のタイムゾーンで、強震モニタのデータが取得できない問題に対応しました。次のバージョンで修正版を公開します。
尚、画面右下に表示される時刻は、引き続き現地のタイムゾーンで表示されます。
次のバージョンの公開時、ぜひご確認ください。
